index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>The Boilerplate</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="module1/_css/pysh_a1.css"/>
```

Comment:
link to the jquery css first and your custom css last so that your styles aren't overwritten